core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackServiceMediaPlayer.java

```diff
@@ -322,7 +322,7 @@ protected void smartMarkAsPlayed(Playable media) {
 
         void onBufferingUpdate(int percent);
 
-        void reloadUI();
+        void reloadUI(boolean onlyMediaSession);
```

Comment:
In any case, I'm working on redesigning the way playback service (and media players) handle episode transitions, so in the near future these semantic changes could become meaningless.